src/Admin/SharedBlockAdmin.php

```diff
@@ -59,7 +59,7 @@ protected function configureQuery(ProxyQueryInterface $query): ProxyQueryInterfa
     protected function configureListFields(ListMapper $list): void
     {
         $list
-            ->addIdentifier('name')
+            ->addIdentifier('name', null, ['route' => ['name' => 'edit']])
```

Comment:
I don't think it's a bad thing to declare the route manually because if not, it rely on the default config of sonata admin. So it could be weird if the user use another default route for all his admins.